packages/widget-chat/src/modules/redux-spark/actions.test.js

```diff
@@ -1,6 +1,7 @@
 /* eslint max-nested-callbacks: ["error", 3] */
+jest.mock(`./spark`);
+const {createSpark} = require(`./spark`);
```

Comment:
Jest says that the babel-jest plugin will do this correctly and not hoist the import over the mock. I couldn't get it to work, so I'm manually requiring.